modules/http/src/test/java/org/mule/module/http/functional/listener/HttpListenerExpectHeaderTestCase.java

```diff
@@ -146,4 +185,15 @@ private void assertFromStream(InputStream inputStream, String expectedInput) thr
         assertThat(new String(actualInput), equalTo(expectedInput));
     }
 
+    protected String getExpectedResponseBody()
+    {
+        return TEST_MESSAGE;
+    }
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> parameters()
+    {
+        return Arrays.asList(new Object[][] {{Boolean.TRUE.toString()},{Boolean.FALSE.toString()}});
```

Comment:
Static imports for Boolean. Also, this should be moved to the top so the parametrisation of the test is clearer.